src/main/java/seedu/tp/flashcard/FlashcardList.java

```diff
@@ -68,6 +68,15 @@ public int getTotalFlashcardNum() {
         return flashcards.size();
     }
 
+    /**
+     * Gets the list of flashcards.
+     * 
+     * @return the list of flashcards
+     */
+    public ArrayList<Flashcard> getFlashcards() {
```

Comment:
Consider using `List<Flashcard>` interface instead of the `ArrayList<Flashcard>` class